docs/conf.py

```diff
@@ -56,15 +56,7 @@
               'sphinx.ext.doctest' ]
 
 doctest_global_setup = '''
-import skfda
```

Comment:
Without these imports I do not pass the local test, there must be a problem in the configuration of doctest on my computer, which does not automatically import the modules of the examples.

Reply:
We should check that, because I can run the tests just fine.